### PR TITLE
fix the horizontal and vertical spacings of members cards

### LINKED ISSUE #1675: [Community] Vertical and horizontal space between member cards should be the same
**Description**
Member cards on /community/members should be equidistant vertically and horizontally.
---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Grumpyyash**
Signed-off-by: Yash Vardhan <yashvardhan513@gmail.com>

Fixed the horizontal and vertical spacing issue in community/members card to make them both equal.

This PR fixes #1675 

equal-spacing

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8f004a146338c6ce294692ffde085391e39c9da7 at: https://60473bf6365228b1382eab26--layer5ng.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8f004a146338c6ce294692ffde085391e39c9da7 at: https://60473c2d365228ac8e2eae8c--layer5ng.netlify.app

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
This doesn't work for 2,3 columns layout.
Moreover, even in this the horizontal spacing is `42.5px` but horizontal is `40px`.

Please update it.

**Comment by l5io** [comment]
🚀 Preview for commit f3ae149f79fd84b58d45e7e4136f79e803af39ce at: https://6047a04fb331ca09614651c6--layer5ng.netlify.app

**Review by Nikhil-Ladha — COMMENTED** [review]
Okay, so I just noticed the overflow is happening due to the svg in the bottom of the page. The `dots` svg, it's not positioned correctly.

**Comment by Nikhil-Ladha** [comment]
The SVG just behind the `Join The Community` button at the bottom of `community/members` page.

**Comment by l5io** [comment]
🚀 Preview for commit 331ed36ba14e5bb12ccdbbf3a44d5809146b3287 at: https://6048fec08c543b00bb99c25d--layer5ng.netlify.app

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
That doesn't mean you will remove the SVG 😂

File: src/sections/Community/Members-grid/index.js [hunk 1]
```diff
@@ -63,17 +63,14 @@ const MembersGrid = props => {
 
               <p className="text-gray">Would you like a member profile?</p>
               <div className="">
-                <VintageBox right={true} vintageOne={true}>
+                <Button primary className="banner-btn memberProfileBtn" title="Let Layer5 Community Managers Know" url="mailto:community@layer5.io?subject=Layer5 Community Membership" external="true">
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Members-grid/index.js:66]: Also, while you are here update the `external` prop to `external={true}`

Comment (Grumpyyash) [reply src/sections/Community/Members-grid/index.js]: Updated!

**Comment by l5io** [comment]
🚀 Preview for commit 125dbd91aab109a7aef4d10287551ef37b302f20 at: https://6049ad2c8fb7042955fa92c1--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**